integration_tests/test_swapping.py

```diff
@@ -0,0 +1,176 @@
+from rendermodules.stack import swap_zs
+from rendermodules.pointmatch.swap_point_match_collections import SwapPointMatchesModule
+
+@pytest.fixture(scope='module')
+def render():
+    render_params['project'] = 'swapping_test'
+    render = renderapi.connect(**render_params)
+    return render
+
+@pytest.fixture(scope='module')
+def source_tspecs():
+    source_ts = [renderapi.tilespec.TileSpec(json=d)
+                for d in swap_z_tspecs1]
+    return source_ts
+
+@pytest.fixture(scope='module')
+def target_tspecs():
+    target_ts = [renderapi.tilespec.TileSpec(json=d)
+                for d in swap_z_tspecs2]
+    return target_ts
+
+@pytest.fixture(scope='module')
+def source_stack(render, source_tspecs):
+    source_stack_name = "source_stack"
+    renderapi.stack.create_stack(source_stack_name, render=render)
+    renderapi.client.import_tilespecs(source_stack_name, source_tspecs, render=render)
+    renderapi.stack.set_stack_state(source_stack_name, 'COMPLETE', render=render)
+
+    yield source_stack_name
+    render.run(renderapi.stack.delete_stack, source_stack_name)
+
+@pytest.fixture(scope='module')
+def target_stack(render, target_tspecs):
+    target_stack_name = "target_stack"
+    renderapi.stack.create_stack(target_stack_name, render=render)
+    renderapi.client.import_tilespecs(target_stack_name, target_tspecs, render=render)
+    renderapi.stack.set_stack_state(target_stack_name, 'COMPLETE', render=render)
+
+    yield target_stack_name
+    render.run(renderapi.stack.delete_stack, target_stack_name)
+
+@pytest.fixture(scope='module')
+def pt_matches1(render):
+    match_collection1 = "collection1"
+    pt_matches = [d for d in swap_pt_matches1]
+    renderapi.pointmatch.import_matches(match_collection1,
+                                        pt_matches,
+                                        render=render)
+    yield match_collection1
+    render.run(renderapi.pointmatch.delete_collection, match_collection1)
+
+
+@pytest.fixture(scope='module')
+def pt_matches2(render):
+    match_collection2 = "collection2"
+    pt_matches = [d for d in swap_pt_matches2]
+    renderapi.pointmatch.import_matches(match_collection2,
+                                        pt_matches,
+                                        render=render)
+    yield match_collection2
+    render.run(renderapi.pointmatch.delete_collection, match_collection2)
+
+
+def test_swap_z_module(render, source_stack, target_stack, source_tspecs, target_tspecs):
+    ex = {
+        "render": render_params,
+        "source_stack": [source_stack],
+        "target_stack": [target_stack],
+        "zValues": [[1015]],
+        "delete_source_stack": "False"
+    }
+
+    mod = swap_zs.SwapZsModule(input_data=ex, args=['--output_json', 'output.json'])
+    mod.run()
+
+    # assert that both the stack exist
+    list_of_stacks = renderapi.render.get_stacks_by_owner_project(host=render.DEFAULT_HOST,
```

Comment:
not sure why you included all the render.DEFAULT_HOST etc... i think that will just fill in by default.   This isn't wrong, just concerned there is something wrong with render-python that i'm not aware of.

Reply:
this usually throws an error for me from the ipython interface.. so adding it for extra caution... I do have the latest version of render-python...

Reply:
okay.. thanks.. i'll take a look at render-python and see if i can figure this out, meanwhile no need to change this.